board.js

```diff
@@ -411,8 +413,8 @@ module.exports = class Board {
           img.height - offsetY - offsetTrimY,
           this.padding - this.panX * this.gridsize,
           this.padding - this.panY * this.gridsize,
-          (img.width * this.backgroundZoom * this.zoom) - scaledOffsetX - scaledOffsetTrimX,
-          (img.height * this.backgroundZoom * this.zoom) - scaledOffsetY - scaledOffsetTrimY,
+          this.dWitdh,
```

Comment:
dWidth

Reply:
100% agree about decoupling the FOW transparency from the grid transparency and keeping default behavior unchanged.

I will implement f in the options parser similar to h  and add a variable.

Thanks for the feedback! :)

Reply:
Great stuff thanks!